File: src/network/LEDChangeModel.java
package network;

import com.google.gson.annotations.SerializedName;

import led.LEDDataBundle;
import procedures.ProcedureType;

public class LEDChangeModel {
	
	@SerializedName(value="procedure")
	public ProcedureType mProcedure;
	
	@SerializedName(value="bundle")
	public LEDDataBundle mBundle;
}

File: src/main/java/network/LEDChangeModel.kt
package network

import com.github.iamniklas.liocore.led.LEDDataBundle
import com.google.gson.annotations.SerializedName
import procedures.ProcedureType

class LEDChangeModel {
    @SerializedName(value = "procedure")
    var mProcedure: ProcedureType? = null

    @SerializedName(value = "bundle")
    var mBundle: LEDDataBundle? = null
}